Makefile

```diff
@@ -0,0 +1,43 @@
+# These variables follow the naming convention from the GNU Make documentation
+# but their defaults correspond to the rest of the code (note that changing
+# libexecdir here wouldn't affect the default path for the channel plugin
+# directory used by the dameon for example).
+#
+# https://www.gnu.org/software/make/manual/html_node/Directory-Variables.html
+prefix     = /usr
+sbindir    = $(prefix)/sbin
+libexecdir = $(prefix)/libexec
+datadir    = $(prefix)/share
+sysconfdir = /etc
```

Comment:
Maybe use [Conditional Assignment](https://www.gnu.org/software/make/manual/make.html#Conditional-Assignment) for `prefix` and `sysconfdir` to allow overrides from environment variables.

(inb4: just use `DESTDIR`)

Reply:
> just use `DESTDIR`

I don't get what this is trying to say.

Reply:
> > just use `DESTDIR`
> 
> I don't get what this is trying to say.

That `DESTDIR` prefixes the `prefix` and that one could, at least partially, alter the either `prefix` or `sysconfdir` by a `DESTDIR`. It was no useful comment after all :)